src/main/webapp/search-meal-script.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function searchMeal() {
+    fetch('/search-meal').then(response => response.json()).then((dishes) => 
+    {
+        let container = document.getElementById("dishes-container");
+        container.innerText = "";
+        dishes = dishes ?? [""];
+        dishes.forEach((dish) => {
+            container.appendChild(createListElement(dish));
+        });
+    });
+}
+
+function createListElement(val) {
```

Comment:
Both you and Olga make very similar code, because you both make the very starter basics of your website. After you both submit all you have, take a look to refactor the code and have similar (or "exactly same") functions in some utility "library" that will be common for you two.